File: hw/cañarte/u3/HW28SingletonAndAbstractFactory/GUIFactory/src/main/java/ec/edu/espe/guifactory/view/ClientApp.java

package ec.edu.espe.guifactory.view;

import ec.edu.espe.guifactory.controller.GUIFactory;
import ec.edu.espe.guifactory.controller.Menu;
import ec.edu.espe.guifactory.controller.Button;

/**
 *
 * @author Saray Canarte, Code Creators, DCCO-ESPE
 */
public class ClientApp {
    public static void main(String[] args) {
        GUIFactory aFactory = GUIFactory.getFactory();
      
        Button aButton = aFactory.createButton();
        aButton.caption = "Play";
        aButton.paint(); 

        Menu aMenu = aFactory.createMenu();
        aMenu.caption = "Start";
        aMenu.display();
    }
}

File: hw/Rodriguez/u3/hw28/hw28Python/AbstractFactory/main/ClientApp.py
import sys
import os

sys.path.append(os.path.abspath(os.path.join(os.path.dirname(__file__), '..')))
from controller.GUIFactory import GUIFactory

if __name__ == "__main__":
    a_factory = GUIFactory.get_factory()
    
    a_button = a_factory.create_button()
    a_button.caption = "Play"
    a_button.paint()

    a_menu = a_factory.create_menu()
    a_menu.caption = "Start"
    a_menu.display()
